transformer_engine/pytorch/tensor/float8_tensor.py

```diff
@@ -229,6 +391,8 @@ def _get_quantizer(self) -> Quantizer:
         """
         if self._quantizer is not None:
             return self._quantizer
+        # Now the quantizer for Float8Tensor can be not just Float8Quantizer (delayed scaling)
+        # It can also be current scaling, can we have a better solution here?
```

Comment:
In this case, we need a way to track or somehow infer the type of originally used quantizer. What about adding self._quantizer_type? In this case it is also required to add quantizer type to serialization.

Reply:
To be honest, this case of the "default" quantizer is not really something you would see in the real usage. There it is actually the Quantizer that creates the tensor and attaches itself to it. So I don't think if it is really needed to be handled here (maybe we should just raise an error instead?).